portfolio.html

```diff
@@ -1,6 +1,20 @@
 <html>
 <title>Portfolio</title>
  <head>
+   <!--I have used a separate css file for all my projects so far so maybe it's different
```

Comment:
I didn't realize this was the way we were supposed to make comments and changes this week so I actually edited your file and added comments there.  I will still make comments and suggestions within here now though.

Reply:
That works im going to submit what i have right now then actually look at this further this week to try to pull the css and jquery out and hopefully get it to work.